spec/controllers/application_controller_spec.rb

```diff
@@ -273,4 +273,12 @@ def index
       subject.sign_out
     end
   end
+
+  def expect_user_event(user, event_type)
```

Comment:
Another nit incoming, but thoughts on renaming this to something like `expect_user_events_to_have_been_created(user, event_type)`?